src/components/Drawer/Drawer.tsx

```diff
@@ -97,29 +104,32 @@ const DrawerBox = forwardRef<HTMLDivElement, DrawerProps>(
         {...props}
       >
         <Header>
-          <Inline stretch={StretchEnum.start}>
-            <Padbox paddingSize={SpaceSizes.lg} paddingType="squish">
+          <Padbox
+            paddingSize={SpaceSizes.mdPlus}
+            paddingType={PaddingTypes.squish}
+          >
+            <Inline align="center" stretch={StretchEnum.start}>
               <Inline align="center" gap={SpaceSizes.md}>
                 {hasAdornment && <Adornment>{adornment}</Adornment>}
                 <TitleWrapper>
                   <Title id="ds-drawer-title">{title}</Title>
                 </TitleWrapper>
               </Inline>
-            </Padbox>
-            <CloseButton
-              marginCompensation={SpaceSizes.none}
-              onClose={onClose}
-            />
-          </Inline>
+              <CloseButton
+                marginCompensation={SpaceSizes.md}
+                onClose={onClose}
+              />
+            </Inline>
+          </Padbox>
         </Header>
         <Content
-          paddingSize={SpaceSizes.lg}
+          paddingSize={SpaceSizes.mdPlus}
           paddingType={hasFooter ? 'squish' : 'square'}
         >
           {children}
         </Content>
         {hasFooter && (
-          <Footer paddingSize={SpaceSizes.lg} paddingType="squish">
+          <Footer paddingSize={SpaceSizes.mdPlus} paddingType="squish">
```

Comment:
There is exception for the footer, the padding will be `md` vertically and `mdPlus` horizontally

Reply:
<img width="1314" alt="Screenshot 2023-11-22 at 14 01 19" src="https://github.com/securityscorecard/design-system/assets/18654425/b8ca4446-c257-47d9-bdb5-7903623703f9">